rust/storage/src/s3.rs

```diff
@@ -122,6 +122,7 @@ impl S3Storage {
             .bucket(self.bucket.clone())
```

Comment:
nit: also remove the `#[tracing::instrument(skip(self), level = "trace")]` at the top of the `get_stream_and_e_tag` method?